pybop/plotting/plot2d.py

```diff
@@ -97,22 +102,28 @@ def plot2d(
     # Populate cost matrix
     for i, xi in enumerate(x):
         for j, yj in enumerate(y):
-            costs[j, i] = cost(np.asarray([xi, yj] + additional_values))
+            costs[j, i] = cost_call(
```

Comment:
Is this right? should you apply the transform in this cost call?

Reply:
This is using the partial function `cost_call()` method which has the transform applied.